doc/user/content/sql/subscribe.md

```diff
@@ -113,9 +113,13 @@ with several additional columns that describe the nature of the update:
 </tbody>
 </table>
 
+### `UP TO`
+
+The `UP TO` clause allows specifying a timestamp at which the `SUBSCRIBE` will cease running. If `UP TO` is specified, no rows whose timestamp is greater than or equal to the specified timestamp will be returned.
```

Comment:
It feels very weird to me to document `UP TO` but not `AS OF`. (I know `UP TO` is more general right now because you can use it with any table, not just the hardcoded special tables, but it still feels weird to me.)

Reply:
The same principle applies, so we should either document both or none. Maybe we just clearly mark these as alpha (like we did for the WebSocket API)? My hesitation here is bloating the documentation (and overwhelming users) with nice-to-have features that are implemented to unblock internal work but may not have more generally applicable use cases.